goose3/crawler.py

```diff
@@ -132,14 +132,16 @@ def crawl(self, crawl_candidate):
         return self.process(raw_html, parse_candidate.url, parse_candidate.link_hash)
 
     def process(self, raw_html, final_url, link_hash):
+        #remove footnotes
+        raw_html_no_footnotes = self.formatter.remove_footnotes(raw_html)
```

Comment:
This is changing the `raw_html` value as it was passed in. This is not what we are looking for here. This should be done (one can use the parsers to do this) on the document itself. There may be much easier ways this could be accomplished.

Reply:
I would recommend looking at the `get_formatted_text` function of  `goose3/outputformmatters.py` where you can see that we are cleaning the likely node with the text into the clean format. Perhaps a config option to remove sup tags could be used and checked at this location to determine if sup tags should be removed. It would need to happen before this call: `self.replace_with_text()` which turns sup tags into plain text.

Reply:
Sorry for the spam... 
You could likely do it with something like this (un-tested pseudo code): 
``` python
sups = self.parser.getElementsByTag(doc, tag='sup')
for node in sups:
    self.parser.drop_tag(node)
```